CHANGELOG.md

```diff
@@ -31,6 +31,7 @@
 * Added the `NavigationMapView.showsCongestionForAlternativeRoutes` property to show congestion levels with different colors on alternative route lines. ([#2887](https://github.com/mapbox/mapbox-navigation-ios/pull/2887))
 * Added the `NavigationMapView.navigationMapView(_:didAdd:)` and `NavigationViewController.navigationViewController(_:didAdd:)` delegate methods, which will be called whenever final destination `PointAnnotation` is added to `NavigationMapView` or `NavigationViewController` respectively. ([#2961](https://github.com/mapbox/mapbox-navigation-ios/pull/2961))
 * Added the ability to show specific route leg with color overriding. When specifying the `legIndex` in `NavigationMapView.show(_:legIndex:)`, the route line for the specific route leg would show congestion colors, while other route legs would show `NavigationMapView.routeCasingColor` by default. If not specified, all the route legs would show congestion colors. During turn-by-turn navigation, the default specified route leg is the current route leg. The route leg colors could be overridden by `NavigationMapView.routeCasingColor` and the congestion colors. ([#2833](https://github.com/mapbox/mapbox-navigation-ios/pull/2833))
+* Added the ability to allow developers choose between `NavigationMapView.userCourseView` and map SDK location layer during turn-by-turn navigation. By providing corresponding configuration for `.puck2D` or `.puck3D`, developers could use the location puck from `MapboxMaps` during turn-by-turn navigation. ([#2968](https://github.com/mapbox/mapbox-navigation-ios/pull/2968))
```

Comment:
The developer is the main audience of this changelog, so we can address them directly:

```suggestion
* Added the `NavigationMapView.puckType` property to customize how the user’s current location is displayed on the map. Set this property to `PuckType.puck2D` or `PuckType.puck3D` to use the [location layer](https://docs.mapbox.com/ios/maps/api/10.0.0-beta.20/Enums/LayerType.html#/s:10MapboxMaps9LayerTypeO17locationIndicatoryA2CmF) powered by the Mapbox Maps SDK instead of the default view-backed implementation specified by the `NavigationMapView.userCourseView` property. ([#2968](https://github.com/mapbox/mapbox-navigation-ios/pull/2968))
```